app/controllers/users.ts

```diff
@@ -1,19 +1,19 @@
 const User = require('../models').users;
 const errors = require('../errors');
 
-exports.getUsers = (_, res, next) =>
+exports.getUsers = (_: any, res: { send: (arg0: any) => void; }, next: any) =>
   User.findAll()
-    .then(users => res.send(users))
+    .then((users: any) => res.send(users))
     .catch(next);
 
-exports.createUser = (req, res, next) =>
+exports.createUser = (req: { body: { username: any; }; }, res: { status: (arg0: number) => { end: () => void; }; }, next: any) =>
```

Comment:
there is a type package for the usual express types (@types/express)
[](https://www.npmjs.com/package/@types/express)